lib/internal/util/inspector.js

```diff
@@ -47,6 +48,10 @@ let session;
 function sendInspectorCommand(cb, onError) {
   const { hasInspector } = internalBinding('config');
   if (!hasInspector) return onError();
+  // Do not preview when the permission model is enabled
+  // because this feature require access to the inspector,
```

Comment:
Optional, if you end up needing to run CI again or something. Otherwise, probably not worth the CI and process overhead for a single unimportant one-letter comment change.

```suggestion
  // because this feature requires access to the inspector,
```